lib/macho/macho_file.rb

```diff
@@ -204,13 +202,8 @@ def dylib_id
     # @return [void]
     # @raise [ArgumentError] if `new_id` is not a String
     def dylib_id=(new_id)
-      if !new_id.is_a?(String)
-        raise ArgumentError.new("argument must be a String")
-      end
-
-      if !dylib?
-        return nil
-      end
+      raise ArgumentError.new("argument must be a String") unless new_id.is_a?(String)
```

Comment:
My preference has generally been to turn expressions like this into single lines if they're only *slightly* longer than 80 columns, the rationale being that it keeps the method's exclusion logic as short as possible.

That being said, I don't feel strongly about it as a style - I'm happy to revert it if you prefer the block form (for >  80 lines).



> (And now that you're touching this code, “argument” feels a tad too generic in the error message.)

Agreed, changing it to "new ID" :smile: